### PR TITLE
Call for Submissions

### PR DESCRIPTION
**Pull request by phkrief**

### CODE REVIEW

**Review by autumnfound — APPROVED** [review]
Currently LGTM! Ping for review if you need another review or to merge

**Comment by phkrief** [comment]
Hi @chrisguindon, Could you approve these new changes for our SAAM event? Thx a lot

**Comment by phkrief** [comment]
@ericpoirier, Chris seems to be busy this morning, would you mind merging this PR, please?
Thx a lot

**Review by ericpoirier — APPROVED** [review]

**Pull request merged**
